src/main/java/uk/gov/hmcts/reform/professionalapi/controller/external/OrganisationExternalController.java

```diff
@@ -110,58 +131,82 @@
                     code = 403,
                     message = "Forbidden Error: Access denied"
             ),
             @ApiResponse(
                     code = 404,
-                    message = "Not Found"
+                    message = "No Payment Accounts found with the given Email Address"
+            ),
+            @ApiResponse(
+                    code = 500,
+                    message = "Internal Server Error"
             )
     })
     @GetMapping(
             path = "/pbas",
             produces = APPLICATION_JSON_VALUE
     )
     @Secured({"pui-finance-manager", "pui-user-manager", "pui-organisation-manager", "pui-case-manager"})
-    public ResponseEntity<OrganisationPbaResponse> retrievePaymentAccountByEmail(@NotNull @RequestParam("email") String email, @ApiParam(hidden = true)@OrgId  String orgId) {
+    public ResponseEntity<OrganisationPbaResponse> retrievePaymentAccountByEmail(@NotNull @RequestParam("email") String email, @ApiParam(hidden = true) @OrgId String orgId) {
         //Received request to retrieve an organisations payment accounts by email for external
 
         return retrievePaymentAccountByUserEmail(email, orgId);
     }
 
-
     @ApiOperation(
-        value = "Add an user to an organisation",
-        authorizations = {
-            @Authorization(value = "ServiceAuthorization"),
-            @Authorization(value = "Authorization")
-        }
+            value = "Add a new User to an Organisation",
+            authorizations = {
+                    @Authorization(value = "ServiceAuthorization"),
+                    @Authorization(value = "Authorization")
+            }
     )
     @ApiResponses({
             @ApiResponse(
                     code = 201,
-                    message = "User has been added",
-                    response = OrganisationResponse.class
+                    message = "The new User has been added to the Organisation",
+                    response = NewUserResponse.class
+            ),
+            @ApiResponse(
+                    code = 400,
+                    message = "An invalid request was provided"
             ),
             @ApiResponse(
                     code = 403,
-                    message = "Forbidden"
+                    message = "Forbidden Error: Access denied"
+            ),
+            @ApiResponse(
+                    code = 404,
+                    message = "No Organisation found with the given ID to add new User to"
+            ),
+            @ApiResponse(
+                    code = 409,
+                    message = "A User already exists with the given Email Address"
+            ),
+            @ApiResponse(
+                    code = 500,
+                    message = "Internal Server Error"
             )
     })
     @PostMapping(
             path = "/users/",
             consumes = APPLICATION_JSON_VALUE,
             produces = APPLICATION_JSON_VALUE
     )
+    @ResponseStatus(value = HttpStatus.CREATED)
```

Comment:
what is the significance of this ?

Reply:
If this is not provided then Swagger will provide a default 200 response in the Swagger UI - this annotation tells Swagger than the successful response code is CREATED (201) and not 200 - it is a known issue with Swagger

Reply:
ok fine.